Shared/alerts/DynamicEntityLayerAlertTarget.cpp

```diff
@@ -0,0 +1,197 @@
+/*******************************************************************************
+ *  Copyright 2012-2018 Esri
```

Comment:
```suggestion
 *  Copyright 2012-2024 Esri
```

New files should get new dates. Check throughout please.

Reply:
The new DynamicEntityLayerAlertTarget.h and .cpp files have been updated to 2024. Should the other's I've modified in this PR be updated as well?

Reply:
No, just new files added to source control should have the current year in their copyright.